cookiecutter.json

```diff
@@ -16,6 +16,7 @@
     "Amazon SES",
     "Other SMTP"
   ],
+  "challenge": "n",
```

Comment:
Switch this to be a non-rendered private variable so the users can't accidentally break the challenge by saying no here.

https://cookiecutter.readthedocs.io/en/stable/advanced/private_variables.html#private-variables